src/main/java/org/team1540/robot2018/Tuning.java

```diff
@@ -102,24 +101,24 @@
   @Tunable("[Elevator] Exchange Position")
   public static double elevatorExchangePosition = 500;
 
-  @Tunable("[Elevator] Front Switch Position")
+  @Tunable("[Elevator] Switch Front Position")
   public static double elevatorFrontSwitchPosition = 2900;
 
-  @Tunable("[Elevator] Scale Lower Position")
-  public static double elevatorScalePosition = 7400;
-
-  @Tunable("[Elevator] Low Scale Position")
+  @Tunable("[Elevator] Scale Low Position")
```

Comment:
Since changes to these values have no effect after robot initialization might be better to remove them entirely.

Reply:
We can change this when we redo all of the tags for the tuning values and move some of them to robot preferences.

Reply:
Also, would it make more sense to move tuning values to the subsystem they are relevant to instead of a tuning class?

Reply:
In theory, yes, but it would require adding all of the subsystems to the `AdjustableManager` instead of just `Tuning`, and since those fields have to be public anyway there's no encapsulation benefit.